QUEST_LV_0100.tsv

```diff
@@ -7530,15 +7530,15 @@ QUEST_LV_0100_20150714_007529	Alruida found the gold plate that he hasn't found
 QUEST_LV_0100_20150714_007530	Alruida is happy since all the plates are gathered. Talk to Alruida again.
 QUEST_LV_0100_20150714_007531	Go meet Justas
 QUEST_LV_0100_20150714_007532	Alruida told you to hand over all gold plates that were discovered to Justas. Hand over all the plates to Justas and receive the research records of Demetrius.
-QUEST_LV_0100_20150714_007533	Remaining Danger
-QUEST_LV_0100_20150714_007534	Bidotas at Mine Village is looking for you. Talk with Bidotas.
-QUEST_LV_0100_20150714_007535	Check the movements of Vubbes in Crystal Mine
-QUEST_LV_0100_20150714_007536	Bidotas is worried that more dangerous Vubbes may come down to the village again. Go to the 2nd district in Crystal Mine to check the movements of Vubbes.
-QUEST_LV_0100_20150714_007537	Before reaching the 2nd district, you were attacked by Vubbes. Tell Bidotas about it.
-QUEST_LV_0100_20150714_007538	Talk with Bidotas what to do with more dangerous Vubbes.
-QUEST_LV_0100_20150714_007539	Set the special incense burner of Bidotas
-QUEST_LV_0100_20150714_007540	Bitotas told you that he created the scent that makes Vubbes lethargic and feel fear. Set the incense burner that contains this scent at the 1st floor of 2nd district in Crystal Mine.
-QUEST_LV_0100_20150714_007541	You've set the incense burner well. Report to Bidotas.
+QUEST_LV_0100_20150714_007533	$Remaining Danger
+QUEST_LV_0100_20150714_007534	$Vaidotas in Miners' Village is looking for you. Talk to Vaidotas.
+QUEST_LV_0100_20150714_007535	$Monitoring Vubbe Activity in the Crystal Mine
+QUEST_LV_0100_20150714_007536	$Vaidotas is worried that highly dangerous Vubbes may descend towards the village again. Go to the District 2 in the Crystal Mine and monitor Vubbe activities.
+QUEST_LV_0100_20150714_007537	$Before reaching District 2, you were ambushed by Vubbes. Report to Vaidotas about it.
+QUEST_LV_0100_20150714_007538	$Discuss with Vaidotas on what to do with the highly dangerous Vubbes.
+QUEST_LV_0100_20150714_007539	$Set the Vaidotas's Special Incense Burner
+QUEST_LV_0100_20150714_007540	$Vaidotas told you that he created a scent that makes the Vubbes lethargic and feel fear. Set the Incense Burner containing this scent at District 2 in Crystal Mine 1F.
+QUEST_LV_0100_20150714_007541	$You've set the Incense Burner well. Report back to Vaidotas.
 QUEST_LV_0100_20150714_007542	The guard, Era seems to have troubles due to Pantos. Talk to the guard, Era.
```

Comment:
The guard, Era seems to have troubles due to Pantos. Talk to the guard, Era. -> The guard, Era, seems to have troubles due to Pantos. Talk to him.

Missing comma after the first Era and change the last part to "him".